infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -187,6 +187,12 @@ private void initializeMetadata(TxnContext txn, LogReplicationSession session, b
                 ReplicationStatus defaultSinkStatus = ReplicationStatus.newBuilder()
                         .setSinkStatus(SinkReplicationStatus.newBuilder()
                                 .setDataConsistent(false)
+                                .setReplicationInfo(ReplicationInfo.newBuilder()
```

Comment:
as we are only adding default values, it is enough to construct a ReplicationInfo and directly invoke build().

It will also be useful to add a comment mentioning that this fields are used on the Sink in prior versions, so they must be populated for backward compatibility

Reply:
added a comment, and actually we cannot rely on default value, which will fail the hasField check on client end.